config/default/translations/messages-en.properties

```
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.0.visited_date = Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.1.visited_date_ask_multiple = Visit
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.1.visited_date = Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.1 = Second Visit Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.2 = Third Visit Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.2.visited_date_ask_multiple = Visit
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.2.visited_date = Date
+
+report.pregnancy_home_visit.anc_visits_hf.risk_factors = Risk Factors
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.previous_risk_factors_note = Risk factors:
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.no_previous_risks_note = None
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.first_pregnancy_note = First pregnancy
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.previous_miscarriage_note = Previous miscarriages or stillbirths
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.previous_difficulties_note = Previous difficulties in childbirth
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.more_than_4_children_note = Has delivered four or more children
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.last_baby_born_less_than_1_year_ago_note = Last baby born less than one year ago
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.heart_condition_note = Heart condition
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.asthma_note = Asthma
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.high_blood_pressure_note = High blood pressure
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.diabetes_note = Diabetes
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.new_risks =Patient has additional risk factors that have not  been previously reported 
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.additional_risk_check = Additional factors that could make this pregnancy high-risk
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.additional_risk = Risk factor description.
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.r_risk_factor_present =  Risk factors Present
+
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_next = ANC Visits at Health Facility (Upcoming)
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_next.anc_next_visit_date = Next ANC Visit Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_next.anc_next_visit_date.appointment_date_known = Patient has a specific upcoming ANC appointment datereport.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_next.anc_next_visit_date.appointment_date = Appointment Date
```

Missing newline. Also changed to sentence case... is that correct here?

```suggestion
report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_next.anc_next_visit_date.appointment_date_known = Patient has a specific upcoming ANC appointment date
report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_next.anc_next_visit_date.appointment_date = Appointment date
```